python/paddle/fluid/backward.py

```diff
@@ -1081,6 +1099,16 @@ def _append_backward_ops_(block,
         rename_var_map(dict): used to associate target_grad var name with first grad_op input name.
             Only used in for high order gradient.
     """
+
+    # Build the mapping between the forward op and backward op (Only for auto parallel)
+    def update_distop_context(distop_context, op_grad_to_var,
```

Comment:
maybe name this as `update_dist_op_context`

Reply:
it will be updated in the next pr.